access-token-management/samples/BlazorServer/HostingExtensions.cs

```diff
@@ -62,7 +62,7 @@ public static WebApplication ConfigureServices(this WebApplicationBuilder builde
 
         // registers HTTP client that uses the managed user access token
         builder.Services.AddTransient<RemoteApiService>();
-        builder.Services.AddUserAccessTokenHttpClient("demoApiClient", configureClient: client =>
```

Comment:
There were conflicts between various extension methods. I decided that this parameter doesn't bring much as you can easily chain the subsequent calls.